styles.css

```diff
@@ -93,3 +94,142 @@ header {
   padding: 5px;
   column-gap: 15px;
 }
+
+.works {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  font-family: 'Poppins', sans-serif;
+  background-color: white;
+}
+
+.project {
+  margin: 10px;
+  padding: 20px;
+  border-radius: 4px;
+  text-align: left;
+}
```

Comment:
- [ ] Please use the `flexbox` in your project *container*, because I noticed that your design is not matching with the `Figma` design template. This `project` class is used for your 4 project cards, so it means that if you fix the design problem in one card it will apply to all automatically. It's very important to always adhere to the design because in *the web development* world, **User Experience** matters a lot. Please use the correct `font-family`, `font-weight`, `colour` and spacing in this section's child content and kindly look at the below image to understand where is the problem. 

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-04-28 17-07-36](https://user-images.githubusercontent.com/66476329/235144341-55fe1fc7-35ae-4a5f-a114-7301d30d0512.png)

</td>
<td>

![Screenshot from 2023-04-28 17-10-25](https://user-images.githubusercontent.com/66476329/235144403-09f674c8-7c28-4393-b21e-5fd9ced86a96.png)

</td>
</tr>
<tr>
<td>

![Screenshot from 2023-04-28 17-13-37](https://user-images.githubusercontent.com/66476329/235144831-fc5c8522-3db7-4abe-96bb-1de190894294.png)

</td>
<td>

![Screenshot from 2023-04-28 17-14-52](https://user-images.githubusercontent.com/66476329/235144870-2a1bc94f-4ecb-47f7-b202-bb35198de482.png)

</td>
</tr>
<tr>
<td>

![Screenshot from 2023-04-28 17-15-44](https://user-images.githubusercontent.com/66476329/235145079-2556c9b2-05e0-4641-a58f-c707d995a90b.png)

</td>
<td>

![Screenshot from 2023-04-28 17-17-12](https://user-images.githubusercontent.com/66476329/235145114-0eb68562-177b-4a00-bfd2-328adaaeed1c.png)

</td>
</tr>


</table>